swap/src/bin/asb.rs

```diff
@@ -136,23 +136,23 @@ async fn main() -> Result<()> {
             };
 
             let kraken_rate = KrakenRate::new(config.maker.ask_spread, kraken_price_updates);
+            let namespace = {
+                if testnet {
+                    XmrBtcNamespace::Testnet
+                } else {
+                    XmrBtcNamespace::Mainnet
+                }
+            };
```

Comment:
Pretty sure you can just do this 

```suggestion
            let namespace = if testnet {
                XmrBtcNamespace::Testnet
            } else {
                XmrBtcNamespace::Mainnet
            };
```

Reply:
Thanks!